app/src/main/java/com/x8bit/bitwarden/data/platform/repository/BridgeRepository.kt

```diff
@@ -0,0 +1,27 @@
+package com.x8bit.bitwarden.data.platform.repository
+
+import com.bitwarden.bridge.model.SharedAccountData
+
+/**
+ * Provides an API for querying disk sources required by [BridgeRepositoryImpl].
+ *
+ * Note: this repository should not be injected anywhere other than [BridgeRepositoryImpl].
+ */
+interface BridgeRepository {
```

Comment:
Are we going to change the naming of all this stuff to `AuthenticatorBridge...` after this PR?

Reply:
Yep, absolutely. Here's a ticket for tracking that: https://livefront.atlassian.net/jira/software/c/projects/BITAU/boards/100?selectedIssue=BITAU-172